src/pages/get-started/release-notes.md

```diff
@@ -7,7 +7,23 @@ details: This page lists new features and known issues for each release of Adobe
 
 These release notes describe the latest version of Adobe I/O Events for Adobe Commerce.
 
-## 1.0.0
+## Version 1.0.1
+
+### Release date
+
+February 8, 2023
+
+### Enhancements
+
+*  Added the `storeId`, `websiteId`, and `storeGroupId` to the event payload metadata whenever it is available.
+
+*  Changed the template engine that generates a module with plugins for eventing.
```

Comment:
It's better to say `Changed the template engine that is used in the generating of the module with plugins for eventing.`
As template engine is not generate anything by itself but is only used for the generating process.